addons/rose/addon/styles/hds/overrides.scss

```diff
@@ -53,3 +53,20 @@ form {
     color: inherit;
   }
 }
+
+// TODO: This is a temporary fix that ensures changing the color theme won't alter the dropdown colors.
+.app-frame-sidebar-style-overrides .side-nav-style-overrides {
```

Comment:
Do the style overrides fail if you use only one parent selector? For example what if you delete the `.app-frame-sidebar-style-overrides` class and use only `.side-nav-style-overrides` as the parent selector?

Reply:
correct it does, due to specificity

Reply:
Sorry just tested this in Vercel and it's looking like the 2 selectors are needed so far...